mne/decoding/transformer.py

```diff
@@ -245,6 +245,92 @@ def inverse_transform(self, X, y=None):
         return X.reshape(-1, self.n_channels, self.n_times)
 
 
+class Vectorizer(TransformerMixin):
+    """Class to chain MNE transformer output to scikit-learn estimators.
```

Comment:
Change it to 'Transforms n-dimensional array into 2D array of n_sample by n_features.'